satzung.md

```diff
@@ -77,27 +77,35 @@
 * Die Einladung zur Mitgliederversammlung ist den Mitgliedern in Textform unter Angabe von Ort, Zeit und Tagesordnung mindestens vier Wochen vorher, bei außerordentlichen Mitgliederversammlungen mindestens zwei Wochen vorher zu übersenden.
 * Anträge zur Ergänzung der Tagesordnung können von jedem Mitglied eingebracht werden. Sie müssen eine Woche vor der Versammlung dem Vorstand in Textform vorliegen. Der Versammlungsleiter hat die Ergänzung zu Beginn der Versammlung bekannt zu geben.
 * Eine Übertragung des Stimmrechts auf ein anderes Mitglied ist möglich, wenn die Stimmübertragung zumindest in Textform nachgewiesen wird. Ein Mitglied kann maximal das Stimmrecht von zwei weiteren Mitgliedern übertragen bekommen.
-* Die Mitgliederversammlung ist beschlussfähig, wenn sie ordnungsgemäß einberufen wurde und mindestens drei Siebtel der Stimmberechtigten vertreten sind. Sind weniger als drei Siebtel der stimmberechtigten Mitglieder vertreten, kann die Mitgliederversammlung erneut einberufen werden. Sie ist dann ohne Rücksicht auf die Zahl der vertretenen Mitglieder beschlussfähig. Auf diesen Umstand wird in der Einladung hingewiesen.
+* Die Mitgliederversammlung ist beschlussfähig, wenn sie ordnungsgemäß einberufen wurde.
 * Auf Antrag eines Mitgliedes ist die Abstimmung geheim durchzuführen.
 
 #### § 10 VORSTAND
 * Der Vorstand:
   * führt die Geschäfte des Vereins und fasst die erforderlichen Beschlüsse.
   * führt Aufzeichnungen über Ausgaben und Einnahmen des Vereins und stellt Spendenbescheinigungen aus.
 * Die Mitglieder des Vorstandes sind jeweils einzelvertretungsberechtigt.
-* Der Vorstand besteht aus entweder drei oder fünf Personen und wird auf ein Jahr gewählt. Eine Wiederwahl ist möglich.
+* Der Vorstand besteht aus entweder drei oder fünf Personen.
+* Der Vorstand wird auf ein Jahr gewählt. Eine Wiederwahl ist möglich.
+* Die Amtszeit des Vorstands beginnt mit der Wahl in der Mitgliederversammlung.
+* Nach Ende der Amtszeit bleibt der Vorstand bis zur nächsten Wahl kommissarisch im Amt.
 * Der Vorstand kann sich eine Geschäftsordnung geben, in der insbesondere Geschäftsverteilung innerhalb des Vorstandes und die gegenseitige Vertretung der Vorstandsmitglieder zu regeln ist.
+* Der Vorstand kann beschließen, Mitgliedern (inklusive Vorstands- und Beiratsmitgliedern) eine angemessene Aufwandsentschädigung für geleistete Tätigkeiten zu bezahlen, sofern dies die finanziellen Rahmenbedingungen des Vereins erlauben.  
```

Comment:
Motivation: Mitglidern können wir aktuell eine Aufwandsentschädigung zahlen (z.B. wenn sie einen Workshop abhalten/anleiten) – bei Vorständen können wir das aktuell nicht. Die Möglichkeit, solche Zahlungen auch an Vorstände durchzuführen muss explizit in der Satzung geregelt werden.

Vorstände sollen – genau wie Mitglieder – für Workshops im Interesse der Werkstatt entlohnt werden können.

Es geht hier NICHT um eine Entlohnung der Vorstandstätigkeit.